.github/workflows/push-test.yml

```diff
@@ -8,14 +8,14 @@ jobs:
       uses: actions/checkout@v2
     - name: Install deps
       run: |
-        sudo apt install cmake g++ libaio-dev libgoogle-perftools-dev libunwind-dev clang-format libboost-dev libboost-program-options-dev libmkl-full-dev libcpprest-dev
+        sudo apt install cmake g++ libaio-dev libgoogle-perftools-dev libunwind-dev clang-format libboost-dev libboost-program-options-dev libboost-test-dev libmkl-full-dev libcpprest-dev
```

Comment:
Is adding another dependence essential to this?

Reply:
yes. to use the boost unit test framework to write the unit test